test/README.md

```diff
@@ -13,25 +13,25 @@ GitHub with the `autocrlf` git config flag set to true.
 
 ## Test Directories
 
-|Directory          |Runs on CI     |Purpose        |
-|-------------------|---------------|---------------|
-|abort              |Yes            |Tests for when the  ``` --abort-on-uncaught-exception ``` flag is used.|
-|addons             |Yes            |Tests for [addon](https://nodejs.org/api/addons.html) functionality along with some tests that require an addon to function properly.|
-|addons-napi        |Yes            |Tests for [n-api](https://nodejs.org/api/n-api.html) functionality.|
-|async-hooks        |Yes            |Tests for [async_hooks](https://nodejs.org/api/async_hooks.html) functionality.|
-|cctest             |Yes            |C++ tests that are run as part of the build process.|
-|code-cache         |No             |Tests for a Node.js binary compiled with V8 code cache.|
-|common             |               |Common modules shared among many tests. [Documentation](./common/README.md)|
-|doctool            |Yes            |Tests for the documentation generator.|
-|es-module          |Yes            |Test ESM module loading.|
-|fixtures           |               |Test fixtures used in various tests throughout the test suite.|
-|internet           |No             |Tests that make real outbound connections (mainly networking related modules). Tests for networking related modules may also be present in        other directories, but those tests do not make outbound connections.|
-|known_issues       |Yes            |Tests reproducing known issues within the system. All tests inside of this directory are expected to fail consistently. If a test doesn't fail on certain platforms, those should be skipped via `known_issues.status`.|
-|message            |Yes            |Tests for messages that are output for various conditions (```console.log```, error messages etc.)|
-|parallel           |Yes            |Various tests that are able to be run in parallel.|
-|pseudo-tty         |Yes            |Tests that require stdin/stdout/stderr to be a TTY.|
-|pummel             |No             |Various tests for various modules / system functionality operating under load.|
-|sequential         |Yes            |Various tests that are run sequentially.|
-|testpy             |               |Test configuration utility used by various test suites.|
-|tick-processor     |No             |Tests for the V8 tick processor integration. The tests are for the logic in ```lib/internal/v8_prof_processor.js``` and  ```lib/internal/v8_prof_polyfill.js```. The tests confirm that the profile processor packages the correct set of scripts from V8 and introduces the correct platform specific logic.|
-|v8-updates         |No             |Tests for V8 performance integration.|
+| Directory           | Runs on CI      | Purpose         |
+| ------------------- | --------------- | --------------- |
+| `abort`             | Yes             | Tests for when the  ``` --abort-on-uncaught-exception ``` flag is used. |
```

Comment:
Out of interest, are the added backticks for the `Directory` column because of the linter or a stylistic change?

Reply:
It's a clunky workaround for the linter flagging bare-text _v8_. If it sees _v8_ in backticks, it assumes it's part of code or a directory listing or something like that, but if it sees it bare, it flags it as a prohibited string and assumes you meant _V8_ instead.

I actually *do* think the directory names should be in backticks, but I also acknowledge that's a debatable position. :-D

Reply:
Thanks for the explanation. I'm okay with the backticks, it just wasn't obvious to me why they were added.